### ISSUE TITLE
update one-time setup docs

### ISSUE THREAD

**Issue by sebastian-luna-valero** [bioconda/bioconda-recipes#3674]
Hello,

I have just installed a fresh bioconda setup and I was stuck by following the instructions here:
https://bioconda.github.io/contrib-setup.html

Would it be possible to include somewhere that you need to run `conda build a-recipe` before using `simulate-travis.py` for the first time? This was commented here:
https://github.com/bioconda/bioconda-utils/issues/37

Many thanks,
Sebastian

**Comment by bgruening** [comment]
@sebastian-luna-valero please feel free to add this to the docs.

**Comment by sebastian-luna-valero** [comment]
Thanks @bgruening 

Please see:
https://github.com/bioconda/bioconda-utils/pull/62

and let me know what you think.

**Comment by bgruening** [comment]
Maybe you can explain why ... To setup needed directories etc ... And give an example recipe that is trivial to compile ... Maybe pyaml?
Thanks!!!

**Comment by sebastian-luna-valero** [comment]
I am happy to update the docs as requested but could you please explain why the solution discussed over https://github.com/bioconda/bioconda-recipes/issues/3126 wasn't finally applied? It is not clear to me after reading the relevant threads.

**Comment by ccwang002** [comment]
Is it possible to add extra information when doing cross-platform compilation? I ran into this issue when I tried to build a new package in Linux container on a clean macOS machine. I need to create an empty index for linux-64 platform at my host macOS first to successfully build the package in the docker, which can be done by the following command:

```shell
conda index $CONDA_ROOT/conda-bld/linux-64
```

The command will create the necessary files `repodata.json` and `repodata.json.bz2` with the empty JSON structure mentioned in #3126.

Before having an empty Linux index, I got the HTTP 404 error when I ran `TRAVIS_OS_NAME=linux ./simulate-travis.py --package mypkg` even though I already built mypkg on macOS:

```
BUILD START: mypkg
updating index in: /opt/conda/conda-bld/linux-64
updating index in: /opt/conda/conda-bld/noarch
...
requests.exceptions.HTTPError: 404 Client Error: None for url: file:///opt/host-conda-bld/linux-64/repodata.json.bz2
```

After creating an empty index at linux-64 platform the HTTP 404 error went away. 

Therefore not only one needs to trigger `conda build` at the host platform but also requires all platforms involved to have the index (can be empty) to run the docker test.
